data/human/variants.txt

```diff
@@ -11,6 +11,20 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+ship "Aerie" "Aerie (Derelict)"
```

Comment:
I feel like it might make more sense to put these derelict variants in their own file. They aren't used as variants in the same sense as the rest of the ships in this file. Derelict ships are effectively their own economy, and having to scroll past a bunch of unrelated ship variants to make any potential balance changes to the derelict ship economy in the future would be a pain.